tests/test_unit_test.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import
+from faker import Faker
+
+faker = Faker()
```

Comment:
I can't see if we are utilizing this anywhere. If we are keeping it for future use, let's add a comment with a use case, otherwise, we can remove this from all the places and plan to introduce it in the second phase. Thoughts?